app/assets/javascripts/select2.full.min.js

```diff
@@ -1,2 +1,3 @@
-/*! Select2 4.1.0-rc.0 | https://github.com/select2/select2/blob/master/LICENSE.md */
```

Comment:
I think this should go in vendor/assets/javascripts/

Reply:
It was moved to vendor/assets/javascripts